package.json

```diff
@@ -11,7 +10,7 @@
   "scripts": {
     "pretest": "./node_modules/.bin/coffeelint ./index.coffee ./test",
     "test": "./node_modules/.bin/mocha --compilers coffee:coffee-script/register test/lib.coffee",
-    "prepublish": "./node_modules/.bin/coffee -c index.coffee"
+    "prepublish": "npm run test && ./node_modules/.bin/coffee -c index.coffee"
```

Comment:
Can you change this from `./node_modules/.bin/coffee` to just `coffee` please (the same for mocha/etc above)